### ISSUE TITLE
Final QA and testing

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#292]
They complain there are some bugs here and there, like the upload process failing and then succeeding for the same file. Can we do some manual testing to make sure all is working?

**Comment by zsinnema** [comment]
@neuroBazzi can you ask what url they are using? If it's dev then this can be due to the fact that the dev cluster is (by design) unstable.

**Comment by neuroBazzi** [comment]
yes it's dev, my bad I should have mentioned. Yes I would like to add some hours to do manual testing

**Comment by afonsobspinto** [comment]
@neuroBazzi I think you missed the point @zsinnema was trying to make. We don't know just yet the cause of that unstable behavior on the upload but it's possible that the reason for it relies on the fact that they are using our dev cluster for testing / experimenting.

The dev cluster has limited resources and uses so called "spot vms", those are much more cheaper machines from google cloud with the drawback that can be replaced up to once a day -> making f.e. the upload workflow fail. 

For a more stable experience the client should use our staging cluster (yet to be updated).

With that said, how many hours can we spend on testing?

**Comment by neuroBazzi** [comment]
thanks @afonsobspinto. that was clear, the deployment ticket (that needs estimate too) is to update stage with latest from dev and see if that does the trick, if not, we'll look into it. 
About testing, let's consider only manual testing and propose something that makes sense to us. To me 60 hours for testing and bug fixing seems reasonable, but feel free to suggest another number cc @zsinnema

**Comment by afonsobspinto** [comment]
With that amount of hours I think we can humbly ask for @SimaoBolota-MetaCell help, starting by asking him how much time it would take to test if the acceptance criteria of all the user stories are being met.

I don't believe we have explicit/formalized acceptance criteria but we can probably extrapolate them from the issues

**Comment by neuroBazzi** [comment]
of course
